crates/api_models/src/payments.rs

```diff
@@ -239,16 +244,15 @@ pub struct PaymentsRequest {
     #[schema(value_type = Option<MerchantConnectorDetailsWrap>)]
     pub merchant_connector_details: Option<admin::MerchantConnectorDetailsWrap>,
 
-    /// Allowed Payment Method Types for a given PaymentIntent
-    #[schema(value_type = Option<Vec<PaymentMethodType>>)]
-    pub allowed_payment_method_types: Option<Vec<api_enums::PaymentMethodType>>,
-
     /// Business sub label for the payment
     pub business_sub_label: Option<String>,
 
     /// If enabled payment can be retried from the client side until the payment is successful or payment expires or the attempts(configured by the merchant) for payment are exhausted.
     #[serde(default)]
     pub manual_retry: bool,
+
+    /// metadata that will be deprecated. For now only order details can be passed here.
+    pub metadata: Option<MetadataExternal>,
```

Comment:
will need to add schema here?

probably can specify in the schema that this field is gonna be deprecated